python/neuroglancer/ingest/tasks.py

```diff
@@ -1,66 +1,348 @@
 import json
 import base64
+import re
+import numpy as np
+import itertools
+import h5py
+from tempfile import NamedTemporaryFile
+import blosc
 
-class Task(object):
-    """
-    A task is defined by an arbitrary blob of data that should be understood by the client.
+from base import QUEUE_NAME, PROJECT_NAME, Storage
+from neuroglancer import chunks
+
+class IngestTask(object):
```

Comment:
It might be a nice thing to inherit from a Task interface that way instanceof will work correctly if necessary